security-proxy/targets-mapping.properties

```diff
@@ -7,3 +7,5 @@ geoserver=http://localhost:8380/geoserver/
 header=http://localhost:8280/header/
 mapfishapp=http://localhost:8280/mapfishapp/
 mapstore=http://localhost:8280/mapstore/
+datafeeder=http://localhost:8280/datafeeder/
```

Comment:
this is wrong, unless we can build a webapp to be deployed in the "georchestra" tomcat advised by the doc / ansible playbook, but for now, it's a standalone spring-boot application which needs to run on its own port (8080 by default)/

Reply:
Changed to 8080, but I'm pretty sure it will clash in regular setups where something is already listening on this port ...

Reply:
@landryb advised `8480`

Reply:
it just seems logical to follow other tomcats ports.. but *shrug* :)